docs/docs/aleph_zero/collections.md

```diff
@@ -4,304 +4,479 @@ title: Collections
+This segment explores key storage structures that manage various entities within the Invariant Protocol. These collections play a crucial role in organizing and managing data in a structured manner, enhancing the overall functionality and performance of our contract. Within our collection interface, we enforce a tightly defined set of operations available for all data collections. Each collection implements the same basic methods, allowing for consistent data management regardless of the underlying data structures (vectors or mappings).
 
 ## Positions
 
 ```rust
-#[ink::storage_item]
-#[derive(Debug, Default)]
 pub struct Positions {
     positions_length: Mapping<AccountId, u32>,
     positions: Mapping<(AccountId, u32), Position>,
 }
 ```
 
-| Key                       | Value                                                                               |
-| ------------------------- | ----------------------------------------------------------------------------------- |
-| Unique account identifier | A tuple containing the number of positions (u32) and a vector of `Position` objects |
+The `Positions` struct is designed to manage positions associated with different accounts. It uses a mapping data structure where each position is uniquely identified by the user's address and index. The provided functions allow you to add, update, remove, transfer, and retrieve positions and the number of positions associated with specific addresses.
 
-The `Positions` struct is designed to manage positions associated with different accounts. It uses a mapping data structure, where each account is uniquely identified by its `AccountId`, and a tuple containing the number of positions and a vector of positions is stored as the associated value. The provided functions allow you to add, update, remove, transfer, and retrieve positions associated with specific accounts.
+| Type                                | Key                                                              | Value                                        |
+| ----------------------------------- | ---------------------------------------------------------------- | -------------------------------------------- |
+| Mapping<AccountId, u32>             | User's address.                                                  | Number of the user's positions.              |
+| Mapping<(AccountId, u32), Position> | Tuple containing the user's address and the index of a position. | Position struct holding the position's data. |
 
-This documentation should help users understand the purpose of the `Positions` struct and how to use its functions effectively.
+### Position Storage Standard
 
-**Why Positions are Stored in State Instead of NFTs?**
-
-We have chosen to store positions in the state rather than using NFTs for several reasons:
+Why are positions stored in the state instead of NFTs? We have chosen to store positions in the state rather than using NFTs for several reasons:
```

Comment:
"We have chosen to store positions in the (~~state~~ -> contract state/Invariant contract state) rather than using NFTs for several reasons:"